src/extensibility/ExtensionManager.js

```diff
@@ -789,7 +789,7 @@ define(function (require, exports, module) {
     exports.updateExtensions        = updateExtensions;
     exports.getAvailableUpdates     = getAvailableUpdates;
     exports.cleanAvailableUpdates   = cleanAvailableUpdates;
-    exports.isRegistryObjectUpdated = false;
+    exports.hasDownloadedRegistry = false;
```

Comment:
nit: indentation.
Also, a newline before this line would separate the functions above and this state variable even more.